docs/source/conf.py

```diff
@@ -437,6 +437,8 @@
     "range",
     "range_end",
     "range_start",
+    "device_range_end",
+    "device_range_start",
```

Comment:
Adding new entries in this file is not ok.
Please make sure to properly document these APIs (proper docstring and .rst update) if you want them public or make them private by adding a `_` at the beginning of their name!

Reply:
@albanD thank you for the review! The CI complained that these two are not documented (see below) and suggested I should update references in rst files, this is how I ended up doing this change.

Indeed, the documentation is in place and I don't see how it's different from the other functions in the same file.
What's the right approach to document these functions and make the `make coverage` happy?

---

```
> cd docs && make coverage && cat build/coverage/python.txt
...
Undocumented Python objects
===========================
torch.cuda.nvtx
---------------
Functions:
 * device_range_end
 * device_range_start
```

Reply:
Perhaps, this is the right place https://github.com/pytorch/pytorch/blob/main/docs/source/cuda.rst?plain=1#L132-L142 ?

Now I'm wondering what makes difference for these functions? 🤔

Reply:
> Indeed, the documentation is in place and I don't see how it's different from the other functions in the same file.

Some of the existing APIs are unfortunately not properly documented, but we enforce all new APIs are from now on.